testgrid/config/config.proto

```diff
@@ -52,6 +71,9 @@ message TestGroup {
   // deprecated - always set to true
   bool is_external = 25;
 
+  // Specifies the test name for a test.
+  TestNameConfig test_name_config = 26;
```

Comment:
So right now do we ignore test_name_config? But the intent is that we will stop doing this in the near future?